### PR TITLE
Add a script for exporting Ask data as CSV

### PR DESCRIPTION
**Pull request by higs4281**
Our previous download script was built for the knowledgebase models.
This adds a script, `export_ask_data`, that can be run via `manage.py
runscript export_ask_data` to export a CSV of English and Spanish answer content.

Note: To export Spanish content, which is full of diacritical marks, the script accepts an 
option, `--script-args spanish`, that will dump Spanish answers in a format that will open correctly 
in our versions of Excel.

## Testing
- Run the script for English content and metadata:
  ```
  python cfgov/manage.py runscript export_ask_data
  ```
  A time-stamped CSV, such as "answers_2017-06-27.csv," will be dumped to the project root.

- Run the script for Spanish content:
```
python cfgov/manage.py runscript export_ask_data --script-args spanish
```
A TSV (in UTF-16), "spanish_answers_2017-06-27.tsv," will be dumped.

The need for separate outputs is explained in the script's docstring.

### CODE REVIEW

**Comment by higs4281** [comment]
Holding this for stakeholder feedback.

File: cfgov/ask_cfpb/scripts/export_ask_data.py [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    return html.strip_tags(unescaped).strip()
+
+
+def clean_and_strip_spanish(data):
+    unescaped = html_parser.unescape(data)
+    return html.strip_tags(
+        unescaped).strip().replace(
+        '\t', ' ').replace(
+        '\n', ' ').replace(
+        '\r', ' ')
```
Comment (willbarton) [inline cfgov/ask_cfpb/scripts/export_ask_data.py]: Is there a possibility of these characters needing to be replaced for non-spanish? Or: can the same `clean_and_strip` function be used for both languages? It only seems to be used when `assemble_output(spanish_only=True)`.

File: cfgov/ask_cfpb/scripts/export_ask_data.py [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+            answer.next_step.title
+            if answer.next_step
+            else '')
+        output_rows.append(output)
+    return output_rows
+
+
+def export_questions(spanish_only=False):
+    """
+    A rather ridiculous script for exporting Ask CFPB Answer content.
```
Comment (willbarton) [inline cfgov/ask_cfpb/scripts/export_ask_data.py]: Why don't you tell us how you really feel 😛

File: cfgov/ask_cfpb/scripts/export_ask_data.py [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+    `python cfgov/manage.py runscript export_ask_data`
+
+    CEE staffers use a version of Excel that can't easily import UTF-8
+    non-ascii encodings. Generally the only Ask content that has characters
+    outside the ascii range is Spanish asnwers, so we export the bulk of the
+    data as UTF-8, and Spanish answers as a separate UTF-16 file that our
+    versions of Excel will open with proper diacritical marks.
+    UTF-16 doubles the file size, which can causes Excel performance issues
+    for the full data set. UTF-16 also plays hob with delimiting,
+    pushing text to next cells and making a mess.
```
Comment (willbarton) [inline cfgov/ask_cfpb/scripts/export_ask_data.py]: And now I understand the UTF16 encoding above and the need for the spanish break-out in `assemble_output()`! A++ docstringing.

File: cfgov/ask_cfpb/scripts/export_ask_data.py (hunk 1 above)
Comment (higs4281) [inline cfgov/ask_cfpb/scripts/export_ask_data.py]: the UTF16 encoding was creating massive delimiting errors if I didn't strip out line endings and tabs. This doesn't happen with UTF8.

File: cfgov/ask_cfpb/scripts/export_ask_data.py (hunk 2 above)
Comment (higs4281) [inline cfgov/ask_cfpb/scripts/export_ask_data.py]: I hope the friendliness of the Wagtail interface will make these dump requests obsolete!

File: cfgov/ask_cfpb/scripts/export_ask_data.py [hunk 4]
```diff
@@ -0,0 +1,143 @@
+from __future__ import unicode_literals
+
+import datetime
+
+import HTMLParser
+from django.utils import html
+
+from paying_for_college.csvkit import csvkit
```
Comment (higs4281) [inline cfgov/ask_cfpb/scripts/export_ask_data.py]: agate (and its predecessor csvkit) is a heavy package with lots of dependencies, some of which can have version clashes with our cfgov requirements. So I couldn't justify including it.
PFC borrowed the unicode-aware readers and writers from csvkit, which are great line cooks for unicode sandwiches.
If we ever get to python 3, this issue goes away.

I'll trap for existence of paying_for_college in INSTALLED_APPS.

Comment (willbarton) [inline cfgov/ask_cfpb/scripts/export_ask_data.py]: > ~~If~~**WHEN** we ever get to python 3, the need for csvkit should go away.

Fixed that for you 😄

**Pull request merged**